tests/integration_tests/performance/test_boottime.py

```diff
@@ -54,11 +54,6 @@ def test_no_boottime(test_microvm_with_api):
     assert not timestamps
 
 
-@pytest.mark.skipif(
-    global_props.cpu_codename == "INTEL_SKYLAKE"
-    and global_props.host_linux_version == "5.10",
-    reason="perf regression under investigation",
-)
```

Comment:
nit: It would be nice to have a simple description about what was the root cause of the regression in commit message.